Enterprise-CMCS/macpro-mdct-mcr
# CMDCT-2794 (Pt. II): Setup MCR w/ Zustand for State Management

### Description

Follow-up PR for [Part I](https://github.com/Enterprise-CMCS/macpro-mdct-mcr/pull/11518).

This is a stacked PR covers the Admin Banner setup and its corresponding tests.

### Related ticket(s)


CMDCT-2974

---
### How to test

Sign in as an Admin and make sure you're able to create and delete banners under `My Account`

### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
